### ISSUE TITLE
Accissibility of teal buttons and tags

### ISSUE THREAD

**Issue by JMurphyWeb** [emfoundation/ce100-app#688]
https://github.com/emfoundation/ce100-app/pull/671#discussion_r101315407

@fannyBkane said she was going to look into making the contrast between the text and background colour a bit stronger in the above comment.

**Comment by fannyBkane** [comment]
Hey @JMurphyWeb I've been looking at this. There are implications with visual identity, I need a bit of time to think it through, in order to make the least amount of changes. Hope to have an update by tomorrow eop, I hope this works for you!

**Comment by JMurphyWeb** [comment]
Sure @fannyBkane . Please let us know of priority when this comes through. thanks

**Comment by fannyBkane** [comment]
Hi. I've reworked a text/UI colour guide for mobile (it works on desktop too, but I will need to add active states and transitions) https://zpl.io/1nIt9d
Can you let me know if that can fit within the sprint? Based on your feedback I will update the mockups to reflects those new colours. Really appreciate your help and great work on this!

**Comment by JMurphyWeb** [comment]
Thanks @fannyBkane - we will have to give a time estimate on it and then prioritise it. Can do this on monday.

**Comment by fannyBkane** [comment]
Hi @JMurphyWeb could you let me know about this when you can?

**Comment by JMurphyWeb** [comment]
@fannyBkane I asked the question to @asimonw on gitter but I know he has been away. I will copy the question in here.

**Comment by JMurphyWeb** [comment]
@asimonw I would like to spend 2 hours getting as much of these done as possible. I don't think they will all get done in time, although I could be wrong.

Of these, I hope that only the edit icon (pressed) could be an issue.
<img width="217" alt="screen shot 2017-02-27 at 15 23 39" src="https://cloud.githubusercontent.com/assets/12638261/23362940/bf61bdae-fd00-11e6-8b4e-97af99864cbe.png">

The components with blue backgrounds only become awkward when we look at the challenges on mobile view.

**Comment by asimonw** [comment]
@JMurphyWeb That sounds very fair! I think a typical 10% darker for button pressed could be a reasonable temporary fix.

**Comment by JMurphyWeb** [comment]
@fannyBkane i have updated the font and links with a white background only. Can you please review:
- the colours
- the use of underlines (i imagine you will want some removed!!)

And let me know in this issue?

**Comment by asimonw** [comment]
@JMurphyWeb Apologies, I now realise that I said "hover state" in my previous comment, while it's probably an active state pseudo-class which needs to be used, because it says "Button pressed." @fannyBkane Is the hover state still just 10% darker?

**Comment by JMurphyWeb** [comment]
@asimonw I didn't know what to do with hovers so have left them out.

**Comment by fannyBkane** [comment]
@JMurphyWeb @asimonw Please bear in mind the UI update here was only showing states for mobile. As there is no hover state on mobile, I thought we could have an indication that you have pressed a button and that next state is loading. I have called it pressed state, but supposedly it wasn't clear... Please let me know if that makes sense and how you'd call that?
![screen shot 2017-02-28 at 10 32 37](https://cloud.githubusercontent.com/assets/20577339/23401916/5b594eac-fda1-11e6-9ccb-1ffe689f00c9.png)

**Comment by JMurphyWeb** [comment]
@fannyBkane that is fine I think.

Did you see my message about staging app?

**Comment by fannyBkane** [comment]
@JMurphyWeb I did - shall  reply in this thread?

**Comment by fannyBkane** [comment]
@JMurphyWeb  Great. I did, thanks! 

On the explore views (organisations, challenges and insights), can you remove the **underline** on the list item only (keep underline under 'Apple'):

On the explore challenges view can you make the link 'Apple' same light grey (#89a0ae) as other text?

![challenges-mob](https://cloud.githubusercontent.com/assets/20577339/23402454/8bb75d76-fda3-11e6-8c2e-471a2a474047.png)

**Comment by fannyBkane** [comment]
On the 'People' page could you apply the  email style to the phone number ![people-mob](https://cloud.githubusercontent.com/assets/20577339/23402600/2987b35c-fda4-11e6-9ecb-d2fe5e06e4c9.png)

**Comment by fannyBkane** [comment]
On the profile, could you please remove the links 'Add new challenge' and 'Archived challenges'?

 ![profile-mob](https://cloud.githubusercontent.com/assets/20577339/23402685/941769ec-fda4-11e6-98f1-f65ddb093e68.png)

**Comment by JMurphyWeb** [comment]
Thanks i'll look into these now @fannyBkane

**Comment by JMurphyWeb** [comment]
in review now @fannyBkane

**Comment by fannyBkane** [comment]
Hey @JMurphyWeb I have noticed on staging that the text and links colour have changed back to the colours we had before you update today, was it planned at all?

**Comment by JMurphyWeb** [comment]
@fannyBkane it is in review

**Comment by asimonw** [comment]
@JMurphyWeb and @fannyBkane It starts looking great, but I don't understand what happened to all the hover states now.. I know that on mobile, the active state is more useful, but most of the beta testing will probably happen on desktop, so hover states are probably going to be more important than active states. Has this been overlooked or did you have something in mind for this @fannyBkane?

**Comment by JMurphyWeb** [comment]
@asimonw sorry it wasn't clear from the accessibility wireframe what we wanted from hover states. The active states listed only.

**Comment by asimonw** [comment]
Fair point @JMurphyWeb. I did ask about hover states a bit higher up the thread, but I guess it got overlooked. I'll have a chat with @fannyBkane see what we do this late in the game.

**Comment by iteles** [comment]
@asimonw Given that the accessibility issue was added outside of the original scope, our view here should be to do as much as possible in the time we have within this sprint. I can definitely confirm that when estimating this issue and determining it did fit into the tail end of this sprint, we didn't include hover states.
This sprint finished on Monday and we're already running quite a bit over.

@JMurphyWeb @mk4111 I know hover states are causing some issues, so could you please estimate how long this would take to implement, qa and deploy?

**Comment by iteles** [comment]
Confirmed by @asimonw over gitter that this is out of scope 👍

**Comment by asimonw** [comment]
We'll pick this up again in a coming sprint.
